### ISSUE TITLE
li[3](-1.0) and li[3](-1b0) wrong sign

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#2253]
Imported from SourceForge on 2024-07-05 22:34:50
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2016-08-23 21:49:20
Original: https://sourceforge.net/p/maxima/bugs/3201

---

`li[3](-1.0)` and `li[3](-1b0)` return positive values, but I think they're supposed to be negative.

~~~~
(%i1) li[3](-1);
                                    3 zeta(3)
(%o1)                             - ---------
                                        4
(%i2) %, numer;
(%o2)                        - 0.9015426773696957
(%i3) li[3](-1), numer;
(%o3)                        - 0.9015426773696957
(%i4) li[3](-1.0);
(%o4)                         0.9015426773696957
(%i5) li[3](-1b0);
(%o5)                        9.015426773696957b-1
~~~~

It appears that `li[2]` does not have a similar problem.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 22:34:51
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2016-08-29 10:27:27
Original: https://sourceforge.net/p/maxima/bugs/3201/#f754

---

- **status**: open --> closed
- **assigned_to**: Dan Gildea

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 22:34:55
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2016-08-29 10:27:27
Original: https://sourceforge.net/p/maxima/bugs/3201/#2409

---

Fixed in commit  [e698e553f080ad34337c64de91e10683d7aed11b]

Todo: tests in rtest_log.mac should probably have a tolerance.  They pass
for me in cmucl, ecl, gcl, and sbcl.
